src/EVTUI/UI/EditorWindow/EditorWindowViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace EVTUI.ViewModels;
+
+public class EditorWindowViewModel : ViewModelBase
+{
+
+    ////////////////////////////
+    // *** PUBLIC MEMBERS *** //
+    ////////////////////////////
+    public DataManager Config;
+
+    ////////////////////////////
+    // *** PUBLIC METHODS *** //
+    ////////////////////////////
+    public EditorWindowViewModel(DataManager dataManager)
+    {
+        this.Config          = dataManager;
```

Comment:
Whitespace looks a bit big here

Reply:
Ah... that's because it's aligned with another property that isn't part of this PR, but I accidentally put the spacing in this version. I'll cut it down for now.